Update to 4.6.3 (#2358)
ibexa/documentation-developer: docs/update_and_migration/from_4.5/update_from_4.5.md

php bin/console ibexa:elasticsearch:put-index-template --overwrite
php bin/console ibexa:reindex
```

## v4.6.3

### Update notification config

> If you are already customizing the package `ibexa/notifications`, and used SiteAccess aware configuration to change `Notification` subscriptions, your configuration no longer works. You have to manually change your configuration by using new node name `notifier` instead of the old `notifications`.

Author: TODO: Write a better intro of the package, help the reader to know if he uses it.

Packages requiring `ibexa/notifications`:
- [`ibexa/connector-actito`](https://github.com/ibexa/connector-actito/blob/main/composer.json#L18) for [Transactional emails](https://doc.ibexa.co/en/latest/commerce/transactional_emails/transactional_emails/)
- [`ibexa/payment`](https://github.com/ibexa/payment/blob/main/composer.json#L22)
- [`ibexa/user`](https://github.com/ibexa/user/blob/main/composer.json#L31)

(Other packages having it as `require-dev` are ignored).

Author: ```suggestion
The configuration of the package `ibexa/notifications` has changed.
This package is required by other packages, such as `ibexa/connector-actito` for [Transactional emails](https://doc.ibexa.co/en/latest/commerce/transactional_emails/transactional_emails/), `ibexa/payment`, or `ibexa/user`.

If you are already customizing the package `ibexa/notifications` configuration, and used SiteAccess aware configuration to change `Notification` subscriptions, you have to manually change your configuration by using new node name `notifier` instead of the old `notifications`.
```

php bin/console ibexa:elasticsearch:put-index-template --overwrite
php bin/console ibexa:reindex
```

## v4.6.2

#### Database update

Run the following scripts:

=== "MySQL"

>     ``` sql

Reviewer: ```suggestion
    ``` bash
```

We've been using `bash` styling for all SQL scripts in all other upgrade instructions until now. If you think `sql` styling is more fitting, please change it everywhere. Otherwise, use `bash`.

Author: You're right, this isn't SQL

Author: ```suggestion
    ``` bash
```

    mysql -u <username> -p <password> <database_name> < vendor/ibexa/installer/upgrade/db/mysql/ibexa-4.6.1-to-4.6.2.sql
    ```

=== "PostgreSQL"

>     ``` sql

Author: ```suggestion
    ``` bash
```

    psql <database_name> < vendor/ibexa/installer/upgrade/db/postgresql/ibexa-4.6.1-to-4.6.2.sql
    ```

## v4.6.3

> ### Update notification config

Author: ```suggestion
### Notification config update
```

The configuration of the package `ibexa/notifications` has changed.
This package is required by other packages, such as `ibexa/connector-actito` for [Transactional emails](https://doc.ibexa.co/en/latest/commerce/transactional_emails/transactional_emails/), `ibexa/payment`, or `ibexa/user`.

> If you are already customizing the package `ibexa/notifications` configuration, and used SiteAccess aware configuration to change `Notification` subscriptions, you have to manually change your configuration by using new node name `notifier` instead of the old `notifications`.

Reviewer: ```suggestion
If you are customizing the configuration of the `ibexa/notifications` package, and using SiteAccess aware configuration to change the `Notification` subscriptions, you have to manually change your configuration by using the new node name `notifier` instead of the old `notifications`.
```

> For example, the following 4.6.2 config:

Author: ```suggestion
For example, the following v4.6.2 config:
```

> ```yaml

Reviewer 2: Can we use `hl_lines` to highlight the node? Not sure if it works without file includes

Author: Good idea!

```suggestion
```yaml hl_lines="4"
```

ibexa:
    system:
        my_siteacces_name:

>             notifications: # deprecated

Reviewer 2: ```suggestion
            notifications: # old
```
Nitpick: `deprecated` for me means "no longer recommended, but still working - will be removed in the future".

As far I understand this change needs to be done now, so I'd avoid this word.

Author: I was thinking "deprecated in 4.6.2" but this isn't true as we didn't know it will be renamed.

                subscriptions:
                    Ibexa\Contracts\Shipping\Notification\ShipmentStatusChange:
                        channels:
                            - sms
```

becomes the following from 4.6.3:

> ```yaml

Reviewer 2: (if possible) I'd use `hl_lines` here as well.

> becomes the following from 4.6.3:

Author: ```suggestion
becomes the following from v4.6.3:
```
